### DISCUSSION TITLE
Costs

### DISCUSSION THREAD

**Discussion by awood-ops** [Azure/azqr#142] [Q&A] [answered]
Hi,

I'd be interested in getting the costs, but it says that function is disabled. Is there a way of enabling this?

Thanks

**Comment by cmendible** [comment, marked as answer]
Run the scan command with the `-c` flag.

``` shell
azqr scan -c
```

Please use version `v0.21.2` from: https://github.com/Azure/azqr/releases

**Comment by awood-ops** [comment]
Excellent, that worked. Thanks
